### PR TITLE
[Console] remove BC breaking argument

### PR DESCRIPTION
**Pull request by xabbuh**
| Q | A |
| --- | --- |
| Branch? | master |
| Bug fix? | yes |
| New feature? | no |
| BC breaks? | no |
| Deprecations? | no |
| Tests pass? | yes |
| Fixed tickets | https://github.com/symfony/symfony/pull/18999#discussion_r67304012 |
| License | MIT |
| Doc PR |  |

### CODE REVIEW

**Comment by chalasr** [comment]
👍  Good catch, thank you @xabbuh

File: src/Symfony/Component/Console/Helper/QuestionHelper.php [hunk 1]
```diff
@@ -99,8 +99,14 @@ public function setInputStream($stream)
...
-    public function getInputStream($triggerDeprecationError = true)
+    public function getInputStream()
     {
+        $triggerDeprecationError = true;
+
+        if (func_num_args() > 0) {
+            $triggerDeprecationError = func_get_arg(0);
+        }
+
         if ($triggerDeprecationError) {
```
Comment (nicolas-grekas) [inline src/Symfony/Component/Console/Helper/QuestionHelper.php]: what about: `if (0 < func_num_args() && func_get_arg(0)) {`

Comment (xabbuh) [inline src/Symfony/Component/Console/Helper/QuestionHelper.php]: I guess you mean `0 === func_num_args() || func_get_arg(0)`?

Comment (nicolas-grekas) [inline src/Symfony/Component/Console/Helper/QuestionHelper.php]: yes :)

Comment (xabbuh) [inline src/Symfony/Component/Console/Helper/QuestionHelper.php]: fine for me :)

**Comment by nicolas-grekas** [comment]
:+1:

File: src/Symfony/Component/Console/Helper/QuestionHelper.php [hunk 2]
```diff
@@ -99,9 +99,9 @@ public function setInputStream($stream)
      *
      * @return resource
      */
-    public function getInputStream($triggerDeprecationError = true)
+    public function getInputStream()
     {
-        if ($triggerDeprecationError) {
+        if (0 === func_num_args() || func_get_arg(0)) {
             @trigger_error(sprintf('The %s() method is deprecated since version 3.2 and will be removed in 4.0. Use %s:getStream() instead.', __METHOD__, StreamableInputInterface::class), E_USER_DEPRECATED);
```
Comment (nicolas-grekas) [inline src/Symfony/Component/Console/Helper/QuestionHelper.php]: Should be `%s::getStream()`, isn't it? (double colon)

Comment (chalasr) [inline src/Symfony/Component/Console/Helper/QuestionHelper.php]: Sorry... @xabbuh Can you please BTW fix it?

Comment (chalasr) [inline src/Symfony/Component/Console/Helper/QuestionHelper.php]: Same typo on the setter... I will open a PR.

**Comment by nicolas-grekas** [comment]
Thank you @xabbuh.

File: src/Symfony/Component/Console/Helper/QuestionHelper.php (hunk 2 above)
Comment (chalasr) [inline src/Symfony/Component/Console/Helper/QuestionHelper.php]: See #19073

Comment (xabbuh) [inline src/Symfony/Component/Console/Helper/QuestionHelper.php]: @chalasr Sorry, I missed that comment. Thank you for creating the PR.

**Pull request merged**